doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -619,32 +619,44 @@ file is a particular case of a module called a *library file*.
 
 .. cmd:: Declare ML Module {+ @string }
 
-   This commands dynamically loads OCaml plugins dynamically.  The string
-   argument must be a valid `findlib <http://projects.camlcity.org/projects/findlib.html>`
-   plugin name, for example ``coq-core.plugins.ltac``. The first component of
-   the plugin name is a package that has to be in scope of ``findlib``.
+   This commands will load an OCaml "plugin" and its dependencies
+   dynamically. The string argument must be a valid `findlib
```

Comment:
```suggestion
   Load an OCaml "plugin" and its dependencies
   dynamically. :n:`@string` must be a valid `findlib
```

We omit the boilerplate phrase "This command" at the beginning of command descriptions.